src/main/java/org/owasp/wrongsecrets/MvcConfiguration.java

```diff
@@ -40,15 +38,6 @@ public ITemplateResolver springThymeleafTemplateResolver(ApplicationContext appl
         return resolver;
     }
 
-    @Bean
-    public ThymeleafViewResolver viewResolver(SpringTemplateEngine templateEngine) {
```

Comment:
i think this is used together with the spring-dev-tools for live reloading, are you sure we can delete this?

Reply:
I'm sorry. I forgot to convert it to draft.